torch/_torch_docs.py

```diff
@@ -1638,6 +1638,66 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov, r"""
+cov(input, *, correction=1, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given :attr:`input` and weights (optional). The unbiased sample covariance
```

Comment:
I think this needs to describe how the shape of input relates to the mathematics before getting into concepts like "variables"